apps/omg_watcher/lib/omg_watcher/exit_processor/core.ex

```diff
@@ -641,4 +641,32 @@ defmodule OMG.Watcher.ExitProcessor.Core do
 
   defp sla_margin_safe?(exit_processor_sla_margin, min_exit_period_seconds, ethereum_block_time_seconds),
     do: exit_processor_sla_margin * ethereum_block_time_seconds < min_exit_period_seconds
+
+  @doc """
+  Deletes in-flight exits from state and returns deleted exits
+  """
+  @spec delete_in_flight_exits(__MODULE__.t(), list(map)) :: {__MODULE__.t(), list(InFlightExitInfo.t()), list(any())}
+  def delete_in_flight_exits(%__MODULE__{in_flight_exits: ifes} = state, deletions) do
```

Comment:
```suggestion
  def delete_in_flight_exits(state, deletions) do
```
and use `state.ifes`